EIPS/eip-simplify-161.md

```diff
@@ -0,0 +1,308 @@
+---
+eip:
+title: Simplify EIP 161
+description: Simplify EIP 161 and retroactively deprecate unused aspects of it
+author: Peter Davies (@petertdavies)
+discussions-to:
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-02
+requires: 161
+---
+
+
+## Abstract
+
+EIP 161 is overly complex and has a number of edgecases that are poorly documented and tested. This EIP takes advantage of the complete removal of all remaining empty accounts in block 14,049,881 to clarify it and allows implementors to not implement various edgecases that never occured and are not possible in future.
```

Comment:
```suggestion
EIP-161 is overly complex and has a number of edge cases that are poorly documented and tested. This EIP takes advantage of the complete removal of all remaining empty accounts in block 14,049,881 to clarify it, and allows implementors to not implement various edge cases that never occured and are not possible in the future.
```

The Oxford comma is a suggestion, the others to me seem like typos.

Additionally the Motivation section is missing and some of this text seem to belong to it.